lib/components/viewers/trip-viewer.js

```diff
@@ -155,21 +155,20 @@ class TripViewer extends Component {
         {/* Header Block */}
         <div className="trip-viewer-header">
           {/* Back button */}
-          {!hideBackButton && (
-            <div className="back-button-container">
-              <Button bsSize="small" onClick={this._backClicked}>
-                <StyledIconWrapper>
-                  <ArrowLeft />
-                </StyledIconWrapper>
-                <FormattedMessage id="common.forms.back" />
-              </Button>
-            </div>
-          )}
 
-          {/* Header Text */}
-          <HeaderText className="header-text">
-            <FormattedMessage id="components.TripViewer.header" />
-          </HeaderText>
+          <div className="back-button-container">
```

Comment:
<img width="280" height="124" alt="Screenshot 2025-09-05 at 3 00 58 AM" src="https://github.com/user-attachments/assets/7e19d68d-0c43-4c8d-a2f5-85649971d7f5" />
Alignment here is very off

Reply:
Cleaner now! And I fixed the header on mobile

Reply:
<img width="287" height="130" alt="Screenshot 2025-09-19 at 5 40 39 PM" src="https://github.com/user-attachments/assets/2415f330-dbdf-46f4-9404-27d777524937" />
I think it's still not quite right!

Reply:
Is this spacing issue related? <img width="511" height="110" alt="Screenshot 2025-09-19 at 5 41 55 PM" src="https://github.com/user-attachments/assets/27590b92-e7ab-4e35-903f-094cc7e10af8" />